### DISCUSSION TITLE
Welcome to Ignite UI App Builder Discussions!

### DISCUSSION THREAD

**Discussion by kdinev** [IgniteUI/app-builder#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hello, my name is Trent and I'm just getting started with App Builder and have tons of questions, however, only one pressing question.

**Reply by zdrawku** [reply]
Hey @AdamsTechPros! Go ahead and share your question here.

As for all other questions, feel free to submit a support item or ask them trhough issues part of the repository. We have a process in place and following it each of the questions/bugs will be assigned to team member and answered in a timely manner.
